examples/panel/book-recommender/requirements.txt

```diff
@@ -0,0 +1,4 @@
+panel
```

Comment:
also add a requirements.lock.txt, generate it with `pip freeze`

Reply:
[added](https://github.com/ploomber/doc/pull/127/commits/a6d5d38d9db668b88244a7c2f31b5520c91d0a1c)